### PR TITLE
Node exit

### PR DESCRIPTION
**Pull request by Aerilym**

### CODE REVIEW

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

File: apps/staking/components/DevSheet.tsx [hunk 1]
```diff
@@ -55,6 +56,25 @@ export function DevSheet({ buildInfo }: { buildInfo: BuildInfo }) {
...
   }, []);
 
+  const { COMMIT_HASH, COMMIT_HASH_PRETTY } = buildInfo.env;
+
+  const remoteFeatureFlagArray = useMemo(() => (data ? Array.from(data) : []), [data]);
+
+  const textToCopy = useMemo(() => {
+    const enabledFeatureFlags = Object.entries(featureFlags)
+      .filter(([_, enabled]) => enabled)
+      .map(([flag]) => flag);
```
Comment (yougotwill) [inline apps/staking/components/DevSheet.tsx:66]:
```suggestion
      .filter(([, enabled]) => enabled)
```

**Review by yougotwill — CHANGES_REQUESTED** [review]
One small tweak

File: apps/staking/components/DevSheet.tsx (hunk 1 above)
Comment (Aerilym) [reply apps/staking/components/DevSheet.tsx:66]: I assume u mean
```diff
- .filter(([_, enabled]) => enabled)
+ .filter(([, enabled]) => enabled)
```

Comment (Aerilym) [reply apps/staking/components/DevSheet.tsx:66]: made that change

**Pull request merged**